test/utils.test.js

```diff
@@ -0,0 +1,48 @@
+import { findById, createNewElement } from './utils.js';
+import hats from '../../products/hats.js';
```

Comment:
not sure when this changed but still should be :
import hats from '../Products/hats.js';